HLTrigger/JetMET/plugins/BuildFile.xml

```diff
@@ -14,6 +14,7 @@
   <use   name="FWCore/Utilities"/>
   <use   name="RecoMET/METAlgorithms"/>
   <use   name="HLTrigger/HLTcore"/>
+  <use   name="HLTrigger/JetMET"/>
```

Comment:
I wasn't sure what the correct way to do this was, but adding this line seem to make it work.

Without this addition in the BuildFile, when I tested this module (using hlt.py), I received the error message:
----- Begin Fatal Exception 29-May-2015 22:06:04 CEST-----------------------
An exception of category 'PluginNotFound' occurred while
   [0] Constructing the EventProcessor
Exception Message:
Unable to find plugin 'HLTPFJetHbbFilter' in category 'CMS EDM Framework Module'. Please check spelling of name.
----- End Fatal Exception -------------------------------------------------


the configuration is available at:
/afs/cern.ch/user/w/woodson/public/hlt.py

and I tested it with IB CMSSW_7_4_X_2015-05-28-2300 (so actually using PR #9325)

Let me know if there is a more correct way to fix this error

Thanks,

Javier